code/modules/mob/living/simple_animal/guardian/guardian_abilities.dm

```diff
@@ -0,0 +1,733 @@
+			stando.melee_damage_upper = initial(stando.melee_damage_upper)
+			stando.obj_damage = initial(stando.obj_damage)
+			stando.environment_smash = initial(stando.environment_smash)
+			stando.alpha = 255
+			stando.range = initial(stando.range)
+			stando.incorporeal_move = 0
+			stando << "<span class='danger'><B>You switch to combat mode.</span></B>"
+			toggle = FALSE
+		else
+			stando.ranged = 0
+			stando.melee_damage_lower = 0
+			stando.melee_damage_upper = 0
+			stando.obj_damage = 0
+			stando.environment_smash = 0
+			stando.alpha = 45
+			stando.range = 255
+			stando.incorporeal_move = 1
+			stando << "<span class='danger'><B>You switch to scout mode.</span></B>"
+			toggle = TRUE
+	else
+		stando << "<span class='danger'><B>You have to be recalled to toggle modes!</span></B>"
+
+/datum/guardian_abilities/ranged/light_switch()
+	if(stando.see_invisible == SEE_INVISIBLE_MINIMUM)
+		stando << "<span class='notice'>You deactivate your night vision.</span>"
+		stando.see_invisible = SEE_INVISIBLE_LIVING
+	else
+		stando << "<span class='notice'>You activate your night vision.</span>"
+		stando.see_invisible = SEE_INVISIBLE_MINIMUM
+
+/datum/action/innate/snare
+	background_icon_state = "bg_alien"
+
+
+/datum/action/innate/snare/plant
+	name = "Plant Snare"
+	button_icon_state = "set_drop"
+
+/datum/action/innate/snare/plant/Activate()
+	var/mob/living/simple_animal/hostile/guardian/A = owner
+	for(var/datum/guardian_abilities/ranged/I in A.current_abilities)
+		if(I.snares.len <6)
+			var/turf/snare_loc = get_turf(owner.loc)
+			var/obj/effect/snare/S = new /obj/effect/snare(snare_loc)
+			S.spawner = owner
+			S.name = "[get_area(snare_loc)] snare ([rand(1, 1000)])"
+			I.snares |= S
+			owner << "<span class='danger'><B>Surveillance snare deployed!</span></B>"
+		else
+			owner << "<span class='danger'><B>You have too many snares deployed. Remove some first.</span></B>"
+
+/datum/action/innate/snare/remove
+	name = "Remove Snare"
+	button_icon_state = "camera_off"
+
+/datum/action/innate/snare/remove_snare/Activate()
+	var/mob/living/simple_animal/hostile/guardian/A = owner
+	for(var/datum/guardian_abilities/ranged/I in A.current_abilities)
+		var/picked_snare = input(owner, "Pick which snare to remove", "Remove Snare") as null|anything in I.snares
+		if(picked_snare)
+			owner -= picked_snare
+			qdel(picked_snare)
+			owner << "<span class='danger'><B>Snare disarmed.</span></B>"
+
+
+
+/datum/guardian_abilities/ranged/ranged_attack(atom/target_atom)
+	if(istype(., /obj/item/projectile))
+		var/obj/item/projectile/P = .
+		if(stando.namedatum)
+			P.color = stando.namedatum.colour
+
+//magician's fried chicken
+
+/datum/guardian_abilities/fire/handle_stats()
+	stando.melee_damage_lower += 4.5
+	stando.melee_damage_upper += 4.5
+	stando.attack_sound = 'sound/items/Welder.ogg'
+	stando.damage_coeff -= list(BRUTE = 0.3, BURN = 0.3, TOX = 0.3, CLONE = 0.3, STAMINA = 0, OXY = 0.3)
```

Comment:
You should just set the coeff with = because I have no idea what -= will do but it probably doesn't do what you want.